bip-0119.mediawiki

```diff
@@ -710,8 +713,8 @@ for an OP_NOP are a soft fork, so existing software will be fully functional wit
 for mining and block validation. Similar soft forks for OP_CHECKSEQUENCEVERIFY and OP_CHECKLOCKTIMEVERIFY
 (see BIP-0065 and BIP-0112) have similarly changed OP_NOP semantics without introducing compatibility issues.
 
-In contrast to previous forks, OP_CHECKTEMPLATEVERIFY will not make scripts
-valid for policy until the new rule is active.
+In contrast to previous forks, OP_CHECKTEMPLATEVERIFY's implementation does not allow transactions with spending
+scripts using it to be accepted to the mempool or relayed under standard policy until the new rule is active.
```

Comment:
Policy is a node decision and cannot be mandated...

Reply:
i don't think i said mandated?

what would you specifically reword it as?

Reply:
it's clear that it's only under standard policy, not any other policy one might use

Reply:
As a recommendation.

Policy is not a topic for standardization generally.

Reply:
Other BIPs absolutely do discuss introducing new standard output types or script types and standardness rules for them.

Policy very likely _should_ be a topic for BIPs, given that prior activations have had mild bugs with respect to what they permit in the mempool pre-activation.

I agree that in theory there could be BIPs that become final and are "upgraded" to Bitcoin Protocol Consensus Documents, but then there should also be things like Bitcoin Implementation Notes that have things like standardness rules and their importance. E.g., widely adopted standardness rules are relied on to prevent DoS attacks (altho not in this BIP in particular).

I can add language to say "recommended".